.circleci/config.yml

```diff
@@ -220,11 +213,20 @@ jobs:
           name: Run e2e collections test
           command: |
             python -m scripts.run_e2e_tests --skip-install --skip-build --suite="collections" --prod_env
+      - run:
+          name: Run e2e accessibility page test
+          command: |
+            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="accessibility" --prod_env
+      - run:
+          name: Run e2e embedding page test
+          command: |
+            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="embedding" --prod_env
```

Comment:
Why are we combining the accessibility and embedding e2e tests with with the navigation and admin tests? Accessibility and embedding have been a little flaky, so it might be better to keep them separate